lib/core/src/client/preview/start.js

```diff
@@ -77,9 +77,16 @@ export const getContext = (() => {
         addons.setChannel(channel);
       }
     }
-
-    const storyStore = new StoryStore({ channel });
-    const clientApi = new ClientApi({ storyStore, decorateStory });
+    let storyStore;
+    let clientApi;
```

Comment:
any way to make that const again ?
I know that Kotlin can do it. (free ad)

Reply:
I'd wrap it into a function, that way the result of the function can be a const